3570/3/rating_places/app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :place, counter_cache: true
```

Comment:
Why we need this option? `counter_cache: true`?

Reply:
https://guides.rubyonrails.org/association_basics.html#options-for-belongs-to-counter-cache

Reply:
I know what this option for. I ask you, why you need this here? And even if this is needed - why you don't have it on user relation

Reply:
Although the :counter_cache option is specified on the model that includes the belongs_to declaration, the actual column must be added to the associated (has_many) model

Reply:
Based on the comments_count, a new average rating is calculated.

Reply:
> Although the :counter_cache option is specified on the model that includes the belongs_to declaration, the actual column must be added to the associated (has_many) model

Not clear for me what you want to say with this comment